Source/HarmonyPatches/Patches_GUI/GUI_PrisonerTab/Patch_PrisonerTab.cs

```diff
@@ -17,67 +17,67 @@ namespace PrisonLabor.HarmonyPatches.Patches_GUI.GUI_PrisonerTab
     ///     1. string in dev mode indicating percentage of being unwatched before escaping
     ///     2. Recruit option
     /// </summary>
-    [HarmonyPatch(typeof(ITab_Pawn_Visitor), "FillTab")]
-    public static class Patch_PrisonerTab
```

Comment:
So no work and work and recuit option?

Reply:
<img width="282" alt="Screen Shot 2020-07-07 at 1 30 20 PM" src="https://user-images.githubusercontent.com/6241020/86839234-057cb200-c056-11ea-94e8-34c5d48da710.png">

Reply:
no they are there

Reply:
Had to remove the dev mod one to minimize the use of harmony.